### PR TITLE
Remove unused, experimental multi-task components

### PR DESCRIPTION
**Pull request by shadeMe**
## Description

The `MultitaskObjective` and `ClozeMultitask` components are incomplete implementations and are not usable in their current state.

Unsure if we want this PR to target `master` or `v4`.

### Types of change

Removal

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Comment by adrianeboyd** [comment]
There is 0% change anyone was using this, but I would vote to move it to v4.

**Review by polm — APPROVED** [review]
With the error gone this looks fine, especially if it's for v4.

It looks like the tests decided to use the older Ubuntu without pulling in #11928, so if they go green this should be good.

**Comment by shadeMe** [comment]
Sorry about the force-push; the rebasing got messy and I needed to re-create the branch and its commits.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ jobs:
       matrix:
         # We're only running one platform per Python version to speed up builds
         Python36Linux:
-          imageName: "ubuntu-latest"
+          imageName: "ubuntu-20.04"
```
[the remark this answers is not in the document]
Comment (shadeMe) [reply azure-pipelines.yml]: Reverted.

**Pull request merged**